stacks-signer/src/tests/contracts/signers-dkg.clar

```diff
@@ -0,0 +1,37 @@
+;; title: signers-dkg
+;; version:
+;; summary: Signer StackerDB test contract
+;; description:
+
+;; traits
+;;
+(impl-trait 'ST1NXBK3K5YYMD6FD41MVNP3JS1GABZ8TRVX023PT.sip-010-trait-ft-standard.sip-010-trait)
```

Comment:
Is there a reason this code needs to implement SIP-010?  If so, then it's missing the SIP-010 trait methods and deployment will fail.

Reply:
This is removed in my other PR. It is not required as far as I can tell.